### PR TITLE
Added solutions for question in lesson 3

### PR DESCRIPTION
**Pull request by AKuederle**
Moved the answer suggested in #300 to the new format and added the answer for the second question.

### CODE REVIEW

File: _episodes/03-create.md [hunk 1]
```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
 > ~~~
 > {: .bash}
 >
-> Why is it a bad idea to do this?
+> Why is it a bad idea to do this? (Notice here that the `planets` project is now also tracking the entire `mars` repository.)
 > How can Dracula undo his last `git init`?
+> > ## Solution
+> > Git repositories can interfere with each other if they are "nested" in the
+> > directory of another. We therefore advise to create each new git repository in a
```
Comment (rgaiacs) [inline _episodes/03-create.md]: Replace "git" with "Git". See https://github.com/swcarpentry/git-novice/pull/324.

File: _episodes/03-create.md [hunk 2]
```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
...
+> > separate directory. Note that we can track files in directories within a git
```
Comment (rgaiacs) [inline _episodes/03-create.md]: I think that "We therefore advise to create each new git repository in a separate directory." is not super clear. What about

> We therefore advise to create new Git repositories **only** when get something like
>
> ~~~
> $ git status
> ~~~
> {: .bash }
> ~~~
> fatal: Not a git repository (or any parent up to mount point /path/to/working/directory)
> Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
> ~~~
> {: .output }

File: _episodes/03-create.md [hunk 3]
```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
...
+> > repository as shown:
+> >
+> > ~~~
+> > touch moon phobos deimos titan    # create moon files
+> > cd ..                             # return to planets directory
+> > ls moons                          # list contents of the moons directory
+> > git add moons/*                   # add all contents of planets/moons
+> > git status                        # show moons files in staging area
+> > git commit -m "add moon files"    # commit planets/moons to planets git repo
```
Comment (rgaiacs) [inline _episodes/03-create.md]: Replace "git repo" with "Git repository".

File: _episodes/03-create.md [hunk 4]
```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
...
+> > ~~~
+> > {: .bash}
+> >
+> > Similarly, we can ignore (as discussed later) entire directories, such as the moons directory:
```
Comment (rgaiacs) [inline _episodes/03-create.md]: Add backquote surrounding moons.

File: _episodes/03-create.md [hunk 5]
```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
...
+> >
+> > ~~~
+> > echo moons >> .gitignore           # ignore all files within moons
```
Comment (rgaiacs) [inline _episodes/03-create.md]: Replace this with 

~~~
$ nano mars.txt
$ cat .gitignore
~~~

~~~
moons
~~~

This is only style.

File: _episodes/03-create.md [hunk 6]
```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
...
+> > ~~~
+> > {: .bash}
+> >
+> > To recover from this little mistake, Dracula can just remove the `.git`
+> > folder in the moons subdirectory. To do so he can run the following command:
+> >
+> > ~~~
+> > rm -rf .git
```
Comment (rgaiacs) [inline _episodes/03-create.md]: I believe that what we need is `rm -rf moons/.git`.

File: _episodes/03-create.md [hunk 7]
```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
...
+> > ~~~
+> > {: .bash}
+> >
+> > But be careful! Running this command in the wrong directory, you will remove
+> > the entire git-history. Therefore, always check your current directory using the
+> > `pwd`-command.
```
Comment (rgaiacs) [inline _episodes/03-create.md]: Use "command `pwd`".

**Comment by rgaiacs** [comment]
@AKuederle Thanks very much for the pull request.

**Comment by daisieh** [comment]
I think this is good to merge as is; I have some issues with how I'd frame the solution, but that would be better addressed as a separate PR and certainly it's way better to have *a* solution than no solution.

**Comment by AKuederle** [comment]
Let me know, what you want change, and I can prepare a second pull request if you want

**Pull request merged**